src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -514,14 +514,16 @@ class SchedulerActions(
     * Make sure the app is running the correct number of instances
     */
   def scale(driver: SchedulerDriver, app: AppDefinition): Unit = {
-    // FIXME (217): use taskTracker.countRunning
-    val currentCount = taskTracker.appTasksSync(app.id).count(_.launched.isDefined)
+    val tasks = taskTracker.appTasksSync(app.id)
+    val runningCount = tasks.count(_.launched.isDefined)
+    val reserved = tasks.count(t => t.reservationWithVolumes.isDefined && t.launched.isEmpty)
+    val runningReserved = runningCount + reserved
     val targetCount = app.instances
 
-    if (targetCount > currentCount) {
-      log.info(s"Need to scale ${app.id} from $currentCount up to $targetCount instances")
+    if (targetCount > runningReserved) {
```

Comment:
Not sure whether that's correct. Consider the following case:
```
running = 3
reserved = 0
app.instances = 3
```
No we scale down to `instances: 2`. Then we have
```
running = 2
reserved = 1
app.instances = 2
```
So that one task was previously running and is now stale. If we scale up again to ìnstances: 3`, the vals above would end up being
```
runningCount = 2
reserved = 1
runningReserved = 3
targetCount = 3
```
So `targetCount == runningReserved` which brings us into the last branch with `Not scaling`